src/public/index.html

```diff
@@ -91,6 +107,7 @@
     <div id='beta'>*** BETA ***</div>
 </div>
 <div id='auth_block'>
+    <button id="advanced" onclick="toggleAdvanced()">Toggle advanced</button>
```

Comment:
Betyr dette at du har lagt GET-varianten bak en knapp?
Det holder ikke helt, siden hele poenget er at de som kommer fra "public" kun skal komme til GET-varianten...

Jeg ser for meg en egen path, helt ovenfra "samlesiden" som kun gir deg GET-variantene. Og gjerne at den er default... (altså den du treffer når du skriver `api.ndla.no`), og at du må skrive en annen url for å få med skrivevariantene (f.eks. `api.ndla.no/advanced`)

Reply:
Nei, omvendt. Jeg har lagt alle andre bak en egen knapp. Når du går inn på dokumentasjonen på vanlig måte som før vil alt være likt, bortsett fra at bare `GET` endepunktene listes ut. Hvis man vil se på alle kan man trykke på en `toggle advanced` knapp. Da vil siden refreshes, og alle endepunktene listes ut.

Jeg kan gi deg en demo hvis du vil?

Reply:
Oppdatert til å ha en egen path `/advanced` for alle endepunkter, og fjernet `toggle advanced`-knappen